source/documentation/at-rules/css.liquid

```diff
@@ -69,7 +69,7 @@ table_of_contents: true
 
 {{ '## `@media`' | markdown }}
 
-{% # Arguments are (in order): `dart`, `libsass`, `node`, `ruby`, optional feature name, additional details within %}
+{% # Arguments are (in order): `dart`, `libsass`, `node`, `ruby`, optional feature name, `useMarkdown` boolean, additional details within %}
```

Comment:
Unrelated, but what do you think about changing `{% compatibility %}` to accept string arguments instead, and parse them in the TS implementation into a proper object:

```liquid
{% compatibility 'dart:"1.11.0"', 'libsass:false', 'ruby:false', 'feature:"Range Syntax"', 'useMarkdown:false' %}
```

Advantages:

- We can get rid of the comment before each invocation
- No need to specify `null` for unused arguments, not including them should be enough
- If [11ty lands support for kwargs](https://github.com/11ty/eleventy/issues/2679), we can migrate to them with a regex that removes the single quotes

Reply:
Implemented in #59.